crates/load-cargo/src/lib.rs

```diff
@@ -246,7 +246,7 @@ impl ProjectFolders {
 #[derive(Default, Debug)]
 pub struct SourceRootConfig {
     pub fsc: FileSetConfig,
-    pub local_filesets: Vec<usize>,
+    pub local_filesets: Vec<u64>,
```

Comment:
This may look ugly but fst uses u64 as its index type so using usize was, I believe, not the right call in the first place. I am aware that such changes are frowned upon but it is either this or I am going to use some usize::try_from().unwrap() in some places :smile: